### ISSUE TITLE
ERROR: Missing hic_results or bowtie_results files for sample

### ISSUE THREAD

**Issue by jing-wan** [aryeelab/hichipper#27]
hello, I meet a error when I used hichipper. 

This is my hichipper Command Line and error：
```
hichipper --out hichipper_res hichipper_config.yaml
Thu Nov 02 22:05:39 CST 2017: Starting hichipper pipeline v0.6.1
Thu Nov 02 22:05:39 CST 2017: Parsing user parameters
ERROR: Missing hic_results or bowtie_results files for sample; either exclude the sample or manage the file architecture/input

```
and this is my hichipper_config.yaml:
```
peaks:
- ChIP-Seq_human_SRR502135_36_SMC3_bowtie2_merged_hichipper_peaks.narrowPeak
resfrags:
- MboI_resfrag_hg19.bed
hicpro_output:
- hicpro_simplify

```
and this is hicpro_simplify file:

```
hicpro_simplify/bowtie_results/bwt2/sample:
SRR3467175_hg19.bwt2pairs.pairstat

hicpro_simplify/hic_results/data/sample:
SRR3467175_allValidPairs             SRR3467175_hg19.bwt2pairs.REPairs  SRR3467175_hg19.bwt2pairs.SinglePairs
SRR3467175_hg19.bwt2pairs.DEPairs    SRR3467175_hg19.bwt2pairs.RSstat   SRR3467175_hg19.bwt2pairs.validPairs
SRR3467175_hg19.bwt2pairs.DumpPairs  SRR3467175_hg19.bwt2pairs.SCPairs
```

I checked my directory stucture with your manual, but I don't know where is wrong.
could you help me to solve it?
Thank you!

**Comment by caleblareau** [comment]
Could you try updating your install to the most recent version? (V0.7.0)

I think that we resolved this error since 0.6.1

**Comment by jing-wan** [comment]
It works. I updated to v0.7.0. Thanks for your prompt answer.
